src/mongo/client/dbclient_rs_test.cpp

```diff
@@ -98,7 +98,7 @@ class BasicRS : public unittest::Test {
 
 void assertOneOfNodesSelected(MockReplicaSet* replSet,
                               ReadPreference rp,
-                              const std::vector<std::string> hostNames) {
+                              const std::vector<std::string>& hostNames) {
```

Comment:
While you aren't wrong that this is wasteful, this is also in a test. I don't see that it matters.